universal-login-web3/src/ui/react/Web3PickerComponent.tsx

```diff
@@ -26,23 +25,18 @@ export const Web3PickerComponent = ({factories, isVisibleProp, hideModal, setPro
   const [selectedProvider, setSelectedProvider] = useState('UniversalLogin');
 
   return isVisible
-    ? <ModalWrapper>
+    ? <ModalWrapper hideModal={() => console.log('hide modal')} message='This is a test environment running on Ropsten network'>
```

Comment:
We get the hideModal callback in component's props, so we can use`hideModal={hideModal}`